ui/apps/platform/src/Containers/Compliance/Entity/ComplianceEntityCluster.jsx

```diff
@@ -26,15 +25,15 @@ function getResourceTabs({
     isComplianceRouteEnabledForNamespaces,
     isComplianceRouteEnabledForNodes,
 }) {
-    const resourceTabs = [entityTypes.CONTROL];
+    const resourceTabs = ['CONTROL'];
     if (isComplianceRouteEnabledForNamespaces) {
-        resourceTabs.push(entityTypes.NAMESPACE);
+        resourceTabs.push('NAMESPACE');
     }
     if (isComplianceRouteEnabledForNodes) {
-        resourceTabs.push(entityTypes.NODE);
+        resourceTabs.push('NODE');
     }
     if (isComplianceRouteEnabledForDeployments) {
-        resourceTabs.push(entityTypes.DEPLOYMENT);
+        resourceTabs.push('DEPLOYMENT');
     }
     return resourceTabs;
```

Comment:
I will improve this first draft of role-based access control in a contribution that is independent of names.